frontend/src/metabase/visualizations/lib/settings/column.js

```diff
@@ -26,6 +26,7 @@ const DEFAULT_GET_COLUMNS = (series, vizSettings) =>
 
 export function columnSettings({
   getColumns = DEFAULT_GET_COLUMNS,
+  hidden,
```

Comment:
this is functionally the same, but since it's a js file, it plays nicer with typescript by making this property explicit.